src/main/java/com/appland/appmap/gradle/AppMapPluginExtension.java

```diff
@@ -15,7 +15,7 @@
  */
 public class AppMapPluginExtension {
 
-  public static final String DEFAULT_OUTPUT_DIRECTORY = "build/appmap";
+  public static final String DEFAULT_OUTPUT_DIRECTORY = "appmap";
```

Comment:
What motivated this change? Was this necessary to use the older API?

Reply:
It's not necessary, but seems better to programmatically retrieve the build directory instead of hard coding it in case `buildDir` is modified. Switching to the older API made this easy.